packages/js/product-editor/src/components/attribute-control/new-attribute-modal.scss

```diff
@@ -53,13 +53,50 @@
 		display: grid;
 		grid-template-columns: 40% 55% 5%;
 		border-bottom: 1px solid $gray-300;
-	}
-	&__table-row {
+
 		padding: $gap-large 0;
 		td:not(:last-child) {
 			margin-right: $gap;
 		}
 
+		.components-form-token-field {
+			position: relative;
+		}
+
+		.components-form-token-field__help {
+			position: absolute;
+			top: 32px;
+			z-index: 0;
+		}
+
+		.components-form-token-field__input-container {
+			position: absolute;
+			top: 0;
+			min-height: 34px;
```

Comment:
Can we change to 32?

Reply:
Why do you think we should change it to `34px`?

Reply:
I had this in my head but didn't remember why. I did a small research and found some references: https://bootcamp.uxdesign.cc/designing-in-the-8pt-grid-system-f3c1183ea6e8

> The 8pt grid system is a helpful way to manage the spacing and sizing of elements throughout a design. The core principle to realize is that each element and space of the UI can be a multiple of 8. This allows us to space, and size all of our information in a concise way that leaves no guesswork when we are designing.

I'm not sure if we use this in WordPress, but I think in most cases, it won't hurt.

Reply:
We do have the core component to address these issues, but I'm trying to adhere to the current design as well.
It's important for us to find the most appropriate solution, which may not necessarily involve adapting the core component to fit our design. In fact, it may make more sense to update our current design to incorporate the core component.

Reply:
@j111q Any reason why we have 34px height in this case?

<img width="1217" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13437655/4d57dfb7-d5a9-4510-a848-ba60adb89f8c">